src/file/file.controller.ts

```diff
@@ -0,0 +1,50 @@
+import * as express from 'express';
+import { IFile } from './file.interface';
+import { fileModel } from './file.model';
+import { fileService } from './file.service';
+import { storageService } from './storage/storage.service';
+
+export class fileController {
+
+  public static async create(files: IFile[]) {
```

Comment:
Functions should have return value too.
Maybe we should add this to the linter somehow...